sqlx-core/src/pool/mod.rs

```diff
@@ -534,28 +533,6 @@ impl<DB: Database> Pool<DB> {
     }
 }
 
-#[cfg(all(
-    any(
-        feature = "postgres",
-        feature = "mysql",
-        feature = "mssql",
-        feature = "sqlite"
-    ),
-    feature = "any"
-))]
-impl Pool<Any> {
-    /// Returns the database driver currently in-use by this `Pool`.
-    ///
-    /// Determined by the connection URL.
-    pub fn any_kind(&self) -> AnyKind {
-        self.0
-            .connect_options
-            .read()
-            .expect("write-lock holder panicked")
-            .kind()
-    }
-}
```

Comment:
Not a breaking change: this `impl` block hasn't been enabled since drivers were moved out of `sqlx-core`.